src/workflows-app/SubmissionDetails.js

```diff
@@ -400,6 +386,26 @@ export const BaseSubmissionDetails = ({ name, namespace, workspace, submissionId
                             return h(TextCell, [customFormatDuration(paginatedPreviousRuns[rowIndex].duration)]);
                           },
                         },
+                        {
+                          size: { basis: 400, grow: 0 },
+                          field: 'duration',
```

Comment:
Is this `'duration'` is a copypasta artifact intended to be something like `'workflowId'`?

Reply:
D'oh, yes